tests/tests/test_timetable.py

```diff
@@ -84,12 +88,51 @@ def test_conflicts(
             "train_name": "pass",
         }
     ]
-    requests.post(f"{EDITOAST_URL}/timetable/{timetable_id}/train_schedule", json=train_schedule_payload)
-    response = requests.get(f"{EDITOAST_URL}/timetable/{timetable_id}/conflicts/?infra_id={small_infra.id}")
-    assert response.status_code == 200
-    actual_conflicts = {conflict["conflict_type"] for conflict in response.json()}
+    requests.post(
+        f"{EDITOAST_URL}/timetable/{timetable_id}/train_schedule", json=train_schedule_payload
+    ).raise_for_status()
+
+    conflicts_response = requests.get(f"{EDITOAST_URL}/timetable/{timetable_id}/conflicts/?infra_id={small_infra.id}")
+    assert conflicts_response.status_code == 200
```

Comment:
Nit, but you should use `conflicts_response.raise_for_status()` (ditto for the other asserts)

Reply:
🙂 hesitated with this (`== 200` vs. `200 =< ... < 400` + consitency)
One vote, there you go (amended 4th commit again): https://github.com/OpenRailAssociation/osrd/compare/34ba402dfbc926c3a485ff1b0d2a2431c1271daf..08755cd77b29ec02968c6ad59a7b8678f891abce